src/components/StructureExplorer/MoleculeInfo.tsx

```diff
@@ -10,13 +13,31 @@ interface MoleculeInfoProps {
 
 export function MoleculeInfo({ mol }: MoleculeInfoProps): JSX.Element {
   const { setSelectedTitle } = useIdContext();
+  const [showWikiBtn, setShowWikiBtn] = useState(false);
 
   return (
     <div
       className="flex h-[217px] flex-col items-center justify-between overflow-hidden border py-2 px-1 text-xs hover:bg-lightgray"
       onClick={() => setSelectedTitle(mol.code)}
+      onMouseEnter={() => setShowWikiBtn(true)}
+      onMouseLeave={() => setShowWikiBtn(false)}
```

Comment:
Instead of a state, I think you should be able to implement only with [TailwindCSS (group-hover)](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state).